autopush/main.py

```diff
@@ -398,6 +405,12 @@ def endpoint_main(sysargs=None):
     args, parser = _parse_endpoint(sysargs)
     scheme = args.endpoint_scheme or \
         "https" if args.ssl_key else "http"
+    senderid_list = None
+    if args.senderid_list:
+        try:
+            senderid_list = json.loads(args.senderid_list)
+        except (ValueError, TypeError), x:
+            log.err("Invalid JSON specified for senderid_list. Ignoring.", x)
```

Comment:
Hrm, you're right. If there's a bad command line config option, the app should bail and not forgive.